### PR TITLE
upd package.json and add tsconfig.json

### PR DESCRIPTION
**Pull request by joaopfg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "start": "concurrently --kill-others \"rollup -c -w\" \"cd .. && static-server\"",
     "build": "rollup -c",
     "test": "rollup -c && jest",
-    "lint": "eslint ./src/*.js ./test/*.js",
+    "lint": "eslint \"./src/**/*.{js,ts}\" \"./test/**/*.{js,ts}\" && tsc --noEmit",
```
Comment (gkjohnson) [inline package.json:14]: You've updated both the eslint command and added the tsc command here but the eslint config hasn't been updated to support ts files. Lets just add the tsc --noEmit command and leave the eslint command alone here.

You'll also need to add latest typescript to the package.json (npm install typescript) so the tsc command works.

**Comment by gkjohnson** [comment]
Great! Almost there -- you can see in the CI results (and should see by running `npm lint` locally) that there's an error when linting:

```
Error: src/XacroParser.d.ts(3,75): error TS7006: Parameter 'command' implicitly has an 'any' type.
```

Can you fix the problematic command by adding an explicit type to it?

**Comment by gkjohnson** [comment]
This looks good now, thanks! I'll publish a new version this week.

File: package.json [hunk 2]
```diff
@@ -56,6 +56,7 @@
     "rollup": "^1.31.0"
   },
   "dependencies": {
-    "expr-eval": "^2.0.2"
+    "expr-eval": "^2.0.2",
+    "typescript": "^4.5.5"
```
Comment (jtbandes) [inline package.json:60]: Is there any reason for this to be an install/runtime dependency? Seems like it should be only a devDependency.

Comment (gkjohnson) [reply package.json:60]: Yes this was an oversight on my part. Just published v0.3.8 with typescript moved to dev dependencies. Thanks!

Comment (jtbandes) [reply package.json:60]: Awesome, thank you!

**Pull request merged**